test/visual-diff/visual-tests

```diff
@@ -94,6 +94,14 @@
       "url": "examples/visual-tests/amp-layout/amp-layout.amp.html",
       "name": "AMP Layout"
     },
+    {
+      "url": "examples/visual-tests/article-fade-in.amp.html",
+      "name": "amp-fx='fade-in' and amp-fx='fade-in-scroll' Visual tests",
+      "experiments": [
+        "amp-fx-fade-in",
+        "amp-fx-fade-in-scroll"
+      ]
```

Comment:
How does the test know when it's safe to take the snapshot? Since there is a fade effect at play here, add `loading_incomplete_css` and `loading_complete_css` sections to this test. (See comments above for how to use them.)